lib/wwwtxt/page.rb

```diff
@@ -0,0 +1,30 @@
+module WwwTxt
+  class Page
```

Comment:
`Page` is meant to be any kind of single document entity such as a web page, RSS item, PDF file, etc. Not sure if I should rely on updating the initialize to pass in different pieces of data, or if I can use an existing structured format such as [Atom](http://en.wikipedia.org/wiki/Atom_(standard)#Example_of_an_Atom_1.0_feed)